GetF!t/SelectedTabManager.swift

```diff
@@ -1,13 +1,40 @@
 import Foundation
+import SwiftUI
 
 class SelectedTabManager: ObservableObject {
     @Published var selectedTab = 9
+    @Published var titleText: LocalizedStringKey = LocalizedStringProvider.WelcomePage.welcome
+
+    var exerciseName: () {
+        switch selectedTab {
+        case 0: titleText = LocalizedStringProvider.ExercisesNames.squat
+        case 1: titleText = LocalizedStringProvider.ExercisesNames.stepUp
+        case 2: titleText = LocalizedStringProvider.ExercisesNames.burpee
+        default: titleText = LocalizedStringProvider.ExercisesNames.sunSalute
+        }
+    }
+
+    func goToWelcomeView() {
+        selectedTab = 9
+        titleText = LocalizedStringProvider.WelcomePage.welcome
+    }
 
     func goToFirstTab() {
         selectedTab = 0
+        titleText = LocalizedStringProvider.ExercisesNames.squat
     }
 
     func goToNextTab() {
         selectedTab += 1
+        exerciseName
+    }
+
+    func opacity(_ index: Int) -> Double {
+        index == selectedTab ? 0.5 : 0
+    }
+
+    func goTo(_ tab: Int) {
```

Comment:
odstránila by som _.
Obecne je vo swifte trend pomenovávať parametre. Vždy treba myslieť na ergonomiku - ako sa to bude používať ostatným ľuďom? Ako sa to bude čítať?

```swift
goTo(3)
goTo(tab: 3)
```

Podľa mňa je viac vypovedajúce do druhé.

Reply:
done